lint-diff.py

```
@@ -257,7 +294,7 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument(
         "--debug", dest="DEBUG", action="store_true", help="print diagnostic output"
     )
-    parser.add_argument("diff_filename", metavar="diff.txt", default=os.getcwd())
+    parser.add_argument("diff_filename", metavar="diff.txt", default=Path.cwd())
```

Ruff (0.14.7)

Undefined name `Path`

(F821)